api/client/stack/deploy.go

```diff
@@ -181,24 +181,42 @@ func deployServices(
 			cspec.User = *service.User
 		}
 
+		// TODO(nishanttotla): retrieve sendAuth from a flag
+		sendAuth := true
```

Comment:
Currently `sendAuth` is just just set to true, but must be set based on the CLI flag. It's not clear how to retrieve the flag in this case.